packages/parcel-transformer/src/index.ts

```diff
@@ -148,6 +152,11 @@ export default new Transformer<ParcelTransformerOpts>({
 
     asset.setCode(output);
 
+    if (!config.ssr) {
+      const metadata = result?.metadata as CustomBabelFileMetadata;
+      asset.meta.styleRules = metadata.styleRules;
```

Comment:
TODO: since Asset stuff needs to be serialised/de-serialised, not sure if there is less overhead to have a single property rather than an array (e.g. just make it a single string rather than an array)